utils/users.py

```diff
@@ -0,0 +1,45 @@
+import os
+from os.path import abspath, dirname, isfile, join
+from typing import Union
+
+from dotenv import load_dotenv
+from peewee import DatabaseError
+
+from ..models.users import User
+
+path: str = dirname(abspath(__file__))
+env_path: str = join(path, "..", ".env")
+load_dotenv(dotenv_path=env_path)
+db_path: str = join(path, "..", "data", os.getenv("DB_NAME"))
+
+
+class AnonymousUser:
+    """
+    A default user if no user is found in the db.
+
+    Attributes:
+        format: The default display format for an anonymous user.
+    """
+
+    format = 1
+
+
+def get_user(nick: str) -> Union[User, AnonymousUser]:
+    """
+    Gets a user out of the database. If not found, it returns a default user object.
+
+    Args:
+        nick: the name of the user's nick.
+
+    Returns:
+        A User model object or a default AnonymousUser.
+    """
+    user: Union[User, AnonymousUser]
+    try:
+        if not isfile(db_path) or not User.table_exists():
+            raise DatabaseError("Users db and table not created yet.")
```

Comment:
At this point, since this is a utility function, you probably don't need to worry about the db stuff here.  Instead, you can probably catch whatever exception is raised when User() is called without a db backend since you have the safety net of an anonymous user.